.github/workflows/codecov.yaml

```diff
@@ -54,20 +54,17 @@ jobs:
         run: |
           docker compose -f etc/deploy/compose/compose-minio.yaml up -d --wait
 
-      - name: Setup cargo-binstall (Linux)
-        if: runner.os != 'Windows'
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          curl -L --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/cargo-bins/cargo-binstall/main/install-from-binstall-release.sh | bash
+      - uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-llvm-cov
 
-      - name: Install cargo-tarpaulin
-        run:
-          cargo binstall cargo-tarpaulin -y --force
+      - name: Install llvm-tools-preview
+        run: |
+          rustup component add llvm-tools-preview
 
       - name: Test
         # use only one job, trying to reduce memory usage
-        run: cargo tarpaulin --jobs 1 --timeout 300 --out xml --skip-clean --features _test-s3
+        run: cargo llvm-cov --codecov --jobs 1 --features _test-s3 --output-path codecov.json
```

Comment:
Do we really need this `jobs` restriction ?
this worked fine https://github.com/trustification/trustify/pull/1834/files#diff-d968391b3a7edaed2de53a8c60aa2a164c40c9bf62511aee0d95657672245637R68 without causing `OOM`

Reply:
Yes, because otherwise it may run into out of memory issues when linking all binaries at the same time.